railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -1,8 +1,8 @@
 # Be sure to restart your server when you modify this file.
 #
-# This file contains migration options to ease your Rails 6.0 upgrade.
-#
-# Once upgraded flip defaults one by one to migrate to the new default.
+# This file contains explicit, new configuration defaults to ease your Rails 6.0
+# upgrade. After upgrading Rails, uncomment each of these new defaults one at a
+# time and test your application.
```

Comment:
Apps aren’t necessarily required to upgrade each option. This file is more to broadcast where Rails is moving.